package NItdItp;

message TSiteRecommendationSettings {
    optional bool Kill = 1;
    optional float Significance = 2;
    optional int64 TtlDays = 3;
    repeated string Tags = 4;
}

message TSiteRecommendationCategories {
    repeated string Categories = 1;
}

